web/datapipeline/clusters_projector.go

```diff
@@ -103,3 +110,229 @@ func getHanaSIDs(c *cluster.Cluster) []string {
 
 	return sids
 }
+
+// parseClusterDetails parses the cluster details depending on the cluster type
+func parseClusterDetails(c *cluster.Cluster) (json.RawMessage, error) {
+	switch detectClusterType(c) {
+	case models.ClusterTypeHANAScaleUp:
+		fallthrough
+	case models.ClusterTypeHANAScaleOut:
+		return parseHANAClusterDetails(c)
+	default:
+		return json.RawMessage{}, nil
+	}
+}
+
+// parseHANAClusterDetails parses the HANA cluster details
+func parseHANAClusterDetails(c *cluster.Cluster) (json.RawMessage, error) {
+	var sid string
+
+	sids := getClusterSIDs(c)
+	if len(sids) > 0 {
+		sid = sids[0]
+	}
+	nodes := parseClusterNodes(c)
+
+	var systemReplicationMode, systemReplicationOperationMode, secondarySyncState, srHealthState string
+	if len(nodes) > 0 {
+		systemReplicationMode, _ = getHANAAttribute(nodes[0], "srmode", sid)
+		systemReplicationOperationMode, _ = getHANAAttribute(nodes[0], "op_mode", sid)
+		secondarySyncState = getHANASecondarySyncState(nodes, sid)
+		srHealthState = getHANAHealthState(nodes[0], sid)
+	}
+
+	dateLayout := "Mon Jan 2 15:04:05 2006"
+	cibLastWritten, _ := time.Parse(dateLayout, c.Crmmon.Summary.LastChange.Time)
+
+	clusterDetail := &entities.HANAClusterDetails{
+		SystemReplicationMode:          systemReplicationMode,
+		SecondarySyncState:             secondarySyncState,
+		SystemReplicationOperationMode: systemReplicationOperationMode,
+		SRHealthState:                  srHealthState,
+		CIBLastWritten:                 cibLastWritten,
+		StonithType:                    getClusterFencingType(c),
```

Comment:
Maybe better to use `parse` as method prefix in the methods where we parse the original raw data and convert it to the entity.